AllReadyApp/Web-App/AllReady/Areas/Admin/Models/TaskSummaryModel.cs

```diff
@@ -36,5 +37,6 @@ public class TaskSummaryModel
         public List<VolunteerModel> AllVolunteers { get; set; } = new List<VolunteerModel>();
 
         public int AcceptedVolunteerCount => AssignedVolunteers?.Where(v => v.HasVolunteered).Count() ?? 0;
+        public int NumberOfVolunteersSignedUp { get; set; }

```

Comment:
`AllVolunteers` refers to volunteers for the activity, not the task. `AssignedVolunteers` refers to volunteers for the task.  At the moment I cannot recall why I did not use `AssignedVolunteers.Count`, so I have changed it to that for now.